resources/scss/ceres/common/_common.scss

```diff
@@ -6,6 +6,12 @@ body .wrapper-main{
     display: none;
 }
 
+.hidden-sm {

```

Comment:
Bootstrap liefert bereits die Klassen `.hidden-{#bp}-up` und `.hidden-{#bp}-down`. Damit sollte sich dasselbe Ergebnis erreichen lassen

Reply:
hier geht es darum, dass ich es nur in sm verstecken will und ich wollte jetzt nicht hidden-sm-down und dann müsste ich mich wieder darum kümmern, dass es in xs wieder sichtbar ist